ocaml/idl/datamodel.ml

```diff
@@ -1711,6 +1711,61 @@ module Data_source = struct
       ()
 end
 
+module Sr_stat = struct
+  let health = Enum ("sr_health", [
+      "healthy", "Storage is fully available";
+      "recovering", "Storage is busy recovering, e.g. rebuilding mirrors.";
+    ])
+
+  let t =
+    let lifecycle = [Published, rel_kolkata, ""] in
```

Comment:
Is it published or preview here and below?

Reply:
I think if we set it to `Prototyped` it won't show up in the docs. 
@jonludlam would you prefer `SR.probe_ext` to be in the docs for Kolkata or not?

Reply:
I think stuff with Prototyped does appear in the docs - at least it is exposed in the SDK and we additionally mark it as experimental.

Reply:
Hmm I do see some prototyped methods in the docs, like Host.get_management_interface, but all the clustering APIs are missing.

Reply:
You are right, I've changed the lifecycle to say prototyped, we can set it to published in the next release.